space_test.go

```diff
@@ -278,12 +278,13 @@ func minimumRequiredCreateSpace() *app.CreateSpacePayload {
 	}
 }
 
-func CreateSpacePayload(name string) *app.CreateSpacePayload {
+func CreateSpacePayload(name, description string) *app.CreateSpacePayload {
```

Comment:
I wonder if the description of a space should be able to store markup as well just like a comment or a WI. What do you think?